backup_and_restore/disaster_recovery/about-disaster-recovery.adoc

```diff
@@ -23,6 +23,25 @@ This also includes situations where you have lost the majority of your control p
 +
 If applicable, you might also need to xref:../../backup_and_restore/disaster_recovery/scenario-3-expired-certs.adoc#dr-recovering-expired-certs[recover from expired control plane certificates].
 +
+[IMPORTANT]
+====
+You should only restore to an old etcd state when your etcd is not functioning and your kube-apiserver is inaccessible because etcd is down.
+If you can reach the kube-apiserver and get etcd content back, doing an etcd restore will almost certainly make your situation worse, not better.
+
+Disaster recovery will take your cluster back in time and all clients will experience a conflicting, parallel history.
```

Comment:
Can you explain a little more about what the parallel history means? How do clients recover from this?

Reply:
A parallel history works like this.

Etcd keeps an increasing resourceVersion in kube API resources. This means that a given resource version implies a total state of all kube API resources.

So when you restore etcd a resourceVersion, which should never be replayed,  is replayed.  In a simple view, it could look like this

1. rv=10, pod/A is created
2. etcd backup is taken here at rv=10
3. rv=11, pod/A is scheduled to node/1
4. etcd is restored to rv=10
5. rv=11, pod/A is  scheduled to node/2

from a client perspective, which rv=11 is "real"?  we have two conflicting histories.